src/components/accordion/accordionItem.scss

```diff
@@ -34,19 +39,13 @@
     padding: 24px 16px;
 
     &:hover {
-      background: var(--kd-color-background-accent-subtle);
+      background-color: var(--kd-color-background-accent-subtle);
+      outline: 0;
     }
-
-    &:focus {
+    &:focus,
+    &:focus-visible {
       border: 2px solid var(--kd-color-border-focus);

```

Comment:
Use `outline` instead. The border is causing everything to jiggle. 

Similar issue with the black border on hover. If you want to add a border on hover, I suggest having it there by default, just transparent, and add the color on hover.